app/src/androidTest/java/org/mozilla/fenix/ui/SettingsSearchTest.kt

```diff
@@ -220,4 +220,59 @@ class SettingsSearchTest {
             verifyVoiceSearchButtonVisibility(false)
         }
     }
+
+    @Test
+    fun undoDeleteSearchEngineTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSearchSubMenu {
+            verifyEngineListContains("Bing")
+            openEngineOverflowMenu("Bing")
+            clickDeleteSearchEngine()
+            clickUndoSnackBarButton()
+            verifyEngineListContains("Bing")
+        }
+    }
+
+    @Test
+    fun deleteDefaultSearchEngineTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSearchSubMenu {
+            verifyEngineListContains("Google")
+            verifyDefaultSearchEngine("Google")
+            openEngineOverflowMenu("Google")
+            clickDeleteSearchEngine()
+            verifyEngineListDoesNotContain("Google")
+            verifyDefaultSearchEngine("Bing")
+        }
+    }
+
+    @Test
+    fun deleteAllSearchEnginesTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSearchSubMenu {
+            deleteMultipleSearchEngines(
+                "Google",
+                "Bing",
+                "Amazon.com",
+                "DuckDuckGo",
+                "eBay",
+            )
+            verifyDefaultSearchEngine("Wikipedia")
+            verifyThreeDotButtonIsNotDisplayed("Wikipedia")
+            openAddSearchEngineMenu()
+            verifyAddSearchEngineListContains(
+                "Google",
+                "Bing",
+                "Amazon.com",
+                "DuckDuckGo",
+                "eBay",
+            )
+        }
```

Comment:
Can we add a comment for each test here that these are expected `en-us` defaults?

Reply:
Done !